config/config.txt

```diff
@@ -187,6 +187,9 @@ ALLOW_HOLIDAYS
 ##Uncomment to show the names of the admin sending a pm from IRC instead of showing as a stealthmin.
 #SHOW_IRC_NAME
 
+##Defines the ticklimit for subsystem initialization. Lower makes world start smoother. Higher makes it faster
+TICK_LIMIT_MC_INIT 500
```

Comment:
Add note stating that a good testing value is `500` and a good production value is `98` (or state that went left commented out it defaults to a good production value)

Also specify unit, its percents, so just before the first `.` add `(In percents of a byond tick)` (I always specify byond tick when I can because so many things use tick to specify different units of time it's best to clear up confusion.)